src/base-tailwind.css

```
@@ -109,9 +112,61 @@
+    --color-sidebar-foreground: var(--sidebar-foreground);
+    --color-sidebar-primary: var(--sidebar-primary);
+    --color-sidebar-primary-foreground: var(--sidebar-primary-foreground);
+    --color-sidebar-accent: var(--sidebar-accent);
+    --color-sidebar-accent-foreground: var(--sidebar-accent-foreground);
+    --color-sidebar-border: var(--sidebar-border);
+    --color-sidebar-ring: var(--sidebar-ring);
+    --color-tertiary: var(--dokan-button-tertiary-background-color, #00000000);
+    --font-sans: var(--font-sans);
+    --font-serif: var(--font-serif);
+    --font-mono: var(--font-mono);
+    --color-tertiary:                                         var(--dokan-button-tertiary-background-color, #00000000);
```

Stylelint (17.6.0)

[error] Unexpected duplicate "--color-tertiary" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)